tensorboard/components/tf_ng_tensorboard/header/containers/header.component.ts

```diff
@@ -45,15 +56,23 @@ const selectActivePluginIndex = createSelector(
   styleUrls: ['./header.component.css'],
 })
 export class HeaderComponent {
-  activePluginIndex$ = this.store.pipe(select(selectActivePluginIndex));
-  plugins$ = this.store.pipe(select(selectPlugins));
+  readonly activePlugin$ = this.store.pipe(select(getActivePlugin));
+  readonly activePluginIndex$ = this.store.pipe(
+    select(selectActivePluginIndex)
+  );
+  readonly plugins$ = this.store.pipe(select(selectPlugins));
+  readonly disabledPlugins$ = this.store.pipe(select(selectDisabledPlugins));
 
-  constructor(private store: Store<State>) {}
+  constructor(private readonly store: Store<State>) {}
 
-  onPluginSelectionChanged(index: number) {
+  onPluginSelectionChanged({index}: MatTabChangeEvent) {
```

Comment:
q: What change in the surrounding code forces this to change? It looks
like `onPluginSelectionChanged` is called from the same places in the
same ways before and after this change.

Reply:
Ah, that's because I changed the listener from `selectedIndexChange` to `selectedTabChange`. They had functional difference which escapes me for now.